org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/editor/contentassist/antlr/internal/DFA.java

```diff
@@ -29,8 +29,8 @@ public int predict(IntStream input) throws RecognitionException {
 		}
 	}
 	
-	protected AbstractInternalContentAssistParser getRecognizer() {
-		return (AbstractInternalContentAssistParser) recognizer;
+	protected BaseInternalContentAssistParser<?, ?> getRecognizer() {
```

Comment:
why ?

Reply:
Because the other DFA class (see xtext-eclipse) now extends this DFA class.